### PR TITLE
Allow --verbose for server mode

### PR DESCRIPTION
**Pull request by notmgsk**

### CODE REVIEW

**Comment by stylewarning** [comment]
Print a warning that `--verbose` is not really appropriate for multithreaded server environments at the top of quilc startup output (in server mode only).

File: app/src/entry-point.lisp [hunk 1]
```diff
@@ -373,7 +376,7 @@
            (multiple-value-bind (processed-program statistics)
                (process-program program chip-spec
                                 :protoquil protoquil
-                                :verbose verbose
+                                :verbose (when verbose *human-readable-stream*)
```
Comment (stylewarning) [inline app/src/entry-point.lisp:379]: please use `if` with explicit else-branch

**Review by stylewarning — CHANGES_REQUESTED** [review]
just this and looks GTM

File: app/src/entry-point.lisp (hunk 1 above)
Comment (notmgsk) [reply app/src/entry-point.lisp]: donezo

File: app/src/entry-point.lisp [hunk 2]
```diff
@@ -352,6 +352,9 @@
            (cl-syslog:rfc-log (*logger* :warning "--backend and --output-file/-o options ~
                                                   don't make sense when using server mode. ~
                                                   Ignoring them.")))
+         (when verbose
+           (setf quil::*compiler-noise* *error-output*)
+           (warn "--verbose output is not appropriate for production multithreaded environments, and should b used when debugging and with care"))
```
Comment (stylewarning) [inline app/src/entry-point.lisp:357]: I'm not going to hold this PR up but WARN takes a format string so you

```
"can break lines ~
 like this."
```

**Review by stylewarning — APPROVED** [review]

**Pull request merged**